roles/openshift_node/templates/node.yaml.v1.j2

```diff
@@ -15,7 +15,7 @@ kind: NodeConfig
 kubeletArguments: {{ openshift.node.kubelet_args | to_json }}
 {% endif %}
 masterKubeConfig: system:node:{{ openshift.common.hostname }}.kubeconfig
-{% if openshift.common.use_openshift_sdn %}
+{% if openshift.common.use_openshift_sdn or openshift.common.use_nuage %}
 networkPluginName: {{ openshift.common.sdn_network_plugin_name }}
```

Comment:
Is networkPluginName still needed for nodes? I don't see anything in the templated config that references the node config file anymore? Or am I just missing where this value is being used.